styles/styles.scss

```diff
@@ -339,6 +339,22 @@ p {
     border: 1px solid $black;
     color: $gray;
   }
+
+  .input-group {
+    .input-group-text{
+      background-color: $black;
+      border: 1px solid $black;
+      color: $gray;
+    }
+    input[type=number]::-webkit-inner-spin-button, 
```

Comment:
@andredbsantos While I also don't like the "up/down" arrows on a input, and while I didn't spot any on the design, should we _hide_ these by default every time ? (just making sure since this _is_ a design choice)